crates/op-rbuilder/src/flashtestations/builder_tx.rs

```
@@ -0,0 +1,581 @@
+    fn compute_block_content_hash(
+        transactions: Vec<OpTransactionSigned>,
+        parent_hash: B256,
+        block_number: u64,
+        timestamp: u64,
+    ) -> B256 {
+        // Create ordered list of transaction hashes
+        let transaction_hashes: Vec<B256> = transactions
+            .iter()
+            .map(|tx| {
+                // RLP encode the transaction and hash it
+                let mut encoded = Vec::new();
+                tx.encode_2718(&mut encoded);
+                keccak256(&encoded)
+            })
+            .collect();
+
+        // Create struct and ABI encode
+        let block_data = BlockData {
+            parentHash: parent_hash,
+            blockNumber: U256::from(block_number),
+            timestamp: U256::from(timestamp),
+            transactionHashes: transaction_hashes,
+        };
+
+        let encoded = block_data.abi_encode();
+        keccak256(&encoded)
+    }
+
+    fn simulate_register_tee_service_tx<ExtraCtx: Debug + Default>(
+        &self,
+        ctx: &OpPayloadBuilderCtx<ExtraCtx>,
+        evm: &mut OpEvm<
+            &mut State<StateProviderDatabase<impl StateProvider>>,
+            NoOpInspector,
+            PrecompilesMap,
+        >,
+    ) -> Result<TxSimulateResult, BuilderTransactionError> {
+        let nonce = get_nonce(evm.db_mut(), self.tee_service_signer.address)?;
+
+        let register_tx = self.signed_register_tee_service_tx(
+            self.attestation.clone(),
+            ctx.block_gas_limit(),
+            ctx.base_fee(),
+            ctx.chain_id(),
+            nonce,
+        )?;
+        let ResultAndState { result, state } = match evm.transact(&register_tx) {
+            Ok(res) => res,
+            Err(err) => {
+                if err.is_invalid_tx_err() {
+                    warn!(target: "flashtestations", %err, "register tee service tx failed");
+                    return Ok(TxSimulateResult::default());
```

why is it okay to ignore the error here?

---

they are not fatal errors should be handled separately (nonce, balance, gas issues)